From f97a103e0b551c12993bdf77f53011cf723553a8 Mon Sep 17 00:00:00 2001
From: Stephan Linz <linz@li-pro.net>
Date: Sat, 13 Mar 2021 23:24:11 +0100
Subject: [PATCH 1/3] doc: Share Kconfig documentation between repos

The Kconfig reference is being turned into a separate
shared documentation set, which needs following specific
Zephyr tweaks:

 1. Do not build the Kconfig docs as part of the Zephyr
    documentation.

 2. Do not link the index page of the Kconfig docs in
    the Zephyr documentation.

Signed-off-by: Ulf Magnusson <Ulf.Magnusson@nordicsemi.no>
Signed-off-by: Ioannis Glaropoulos <Ioannis.Glaropoulos@nordicsemi.no>
Signed-off-by: Dominik Ermel <dominik.ermel@nordicsemi.no>
Signed-off-by: Johann Fischer <johann.fischer@nordicsemi.no>
Signed-off-by: Stephan Linz <linz@li-pro.net>
---
 doc/CMakeLists.txt      | 2 +-
 doc/reference/index.rst | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/doc/CMakeLists.txt b/doc/CMakeLists.txt
index ca433184f..743adb388 100644
--- a/doc/CMakeLists.txt
+++ b/doc/CMakeLists.txt
@@ -437,7 +437,7 @@ endif()
 #
 # Dependencies and final targets
 #
-add_dependencies(html content doxy_real_modified_times kconfig devicetree)
+add_dependencies(html content doxy_real_modified_times devicetree)
 
 add_custom_target(
   htmldocs
diff --git a/doc/reference/index.rst b/doc/reference/index.rst
index 399db3916..762d005ed 100644
--- a/doc/reference/index.rst
+++ b/doc/reference/index.rst
@@ -11,7 +11,6 @@ API Reference
    audio/index.rst
    misc/notify.rst
    bluetooth/index.rst
-   kconfig/index.rst
    crypto/index.rst
    devicetree/index.rst
    drivers/index.rst
